x-pack/elastic-agent/pkg/core/logger/logger.go

```diff
@@ -118,7 +118,7 @@ func makeInternalFileOutput(cfg *Config) (zapcore.Core, error) {
 		file.Interval(defaultCfg.Files.Interval),
 		file.RotateOnStartup(defaultCfg.Files.RotateOnStartup),
 		file.RedirectStderr(defaultCfg.Files.RedirectStderr),
-		file.Suffix(defaultCfg.Files.Suffix),
+		file.Suffix(file.SuffixDate),
```

Comment:
where does file come from

Reply:
It is the package `github.com/elastic/beats/v7/libbeat/common/file`. It is the source of functions and settings like the function  `file.RedirectStdErr` above.

Reply:
But my bad, you did not request to set it to date all the time.